### PR TITLE
Update samples to use Color class

### PR DESCRIPTION
**Pull request by Luiy0**
### Description

The following changes were made to samples where applicable:

- Remove unused imports.
- Remove code marked for deprecation related to the use of `Color`.
- Remove the `controls` module requirement in a few samples that do not use or need it.
- Update to build `3702`: Two samples have deprecated code related to `loadStatus`. These have been marked in [spreadsheet](https://esriis-my.sharepoint.com/:x:/g/personal/jen11242_esri_com/EV2Xhb1ciLtJiP0leXtftnoB7cWZLl_qEeOZSsGBxWnG0g?e=Zvxqaq) and will be updated in a separate PR. No other code deprecations found.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

**Comment by Luiy0** [comment]
Hi @Rachael-E, this one's ready for a first review, thanks.

File: geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java [hunk 1]
```diff
@@ -168,9 +170,9 @@ public void start(Stage stage) {
         for (Geometry geometry : convexHullGeometries) {
 
           // create a simple line symbol for the outline of the convex hull graphic(s)
-          SimpleLineSymbol convexHullLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 5);
+          SimpleLineSymbol convexHullLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.RED, 5);
           // Create the simple fill symbol for the convex hull graphic(s)
-          SimpleFillSymbol convexHullFill = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, 0x00000000, convexHullLine);
+          SimpleFillSymbol convexHullFill = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, Color.TRANSPARENT, convexHullLine);
```
Comment (Rachael-E) [inline geometry/convex-hull-list/src/main/java/com/esri/samples/convex_hull_list/ConvexHullListSample.java]: hmm this has raised a clean up we can use: if the style is null, and the color is transparent, then the sample isn't showing any fill. In the samples case, we just need to see an outline of the convex hull. We can tidy this sample instead by only using the SimpleLineSymbol:

```java
Graphic convexHullGraphic = new Graphic(geometry, convexHullLine);
```

File: geometry/convex-hull/src/main/java/com/esri/samples/convex_hull/ConvexHullSample.java [hunk 2]
```diff
@@ -84,14 +85,14 @@ public void start(Stage stage) {
...
+      SimpleMarkerSymbol markerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, Color.RED, 10);
       Graphic inputsGraphic = new Graphic();
       inputsGraphic.setSymbol(markerSymbol);
       graphicsOverlay.getGraphics().add(inputsGraphic);
 
       // create a graphic to show the convex hull as a blue outline
-      SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFF0000FF, 3);
-      SimpleFillSymbol fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, 0x00000000, lineSymbol);
+      SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.BLUE, 3);
+      SimpleFillSymbol fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, Color.TRANSPARENT, lineSymbol);
```
Comment (Rachael-E) [inline geometry/convex-hull/src/main/java/com/esri/samples/convex_hull/ConvexHullSample.java]: same comment here as before, we don't need this fill (consider convex hull operation as draping a rubber band around points. We have other samples that show how to apply a simple marker symbol too)

```java
          case POLYLINE:
          case POLYGON:
            convexHullGraphic.setSymbol(lineSymbol);
            break;

```

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java [hunk 3]
```diff
@@ -87,12 +88,12 @@ public void initialize() {
     graphicsOverlay.getGraphics().add(sectorGraphic);
 
     // create green symbols for each sector output geometry type
-    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0x8800FF00, null);
-    sectorLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0x8800FF00, 3);
-    sectorMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0x8800FF00, 3);
+    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, Color.web("lime", 0.8), null);
```
Comment (Rachael-E) [inline geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]: for readability and reduced repetition, let's use a local variable to store the lime color

```java
    Color transparentLime = Color.web("lime", 0.8);
    sectorFillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, transparentLime, null);
    sectorLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, transparentLime, 3);
    sectorMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, transparentLime, 3);

```

File: geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java [hunk 4]
```diff
@@ -87,12 +88,12 @@ public void initialize() {
...
+    sectorLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("lime", 0.8), 3);
+    sectorMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, Color.web("lime", 0.8), 3);
 
     // create a red dotted outline graph for showing the geodesic ellipse geometry
```
Comment (Rachael-E) [inline geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java]:
```suggestion
    // create a red dotted outline graphic for showing the geodesic ellipse geometry
```

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java [hunk 5]
```diff
@@ -161,14 +161,14 @@ private void handleKeyReleased(KeyEvent keyEvent) {
         case POLYLINE:
           Color polylineColor = colorPicker.getValue();
           if (polylineColor != null) {
-            KmlLineStyle kmlLineStyle = new KmlLineStyle(ColorUtil.colorToArgb(polylineColor), 8);
+            KmlLineStyle kmlLineStyle = new KmlLineStyle(polylineColor, 8);
```
Comment (Rachael-E) [inline kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]:
```suggestion
            var kmlLineStyle = new KmlLineStyle(polylineColor, 8);
```

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java [hunk 6]
```diff
@@ -161,14 +161,14 @@ private void handleKeyReleased(KeyEvent keyEvent) {
...
             kmlStyle.setLineStyle(kmlLineStyle);
           }
           break;
         case POLYGON:
           Color polygonColor = colorPicker.getValue();
           if (polygonColor != null) {
-            KmlPolygonStyle kmlPolygonStyle = new KmlPolygonStyle(ColorUtil.colorToArgb(polygonColor));
+            KmlPolygonStyle kmlPolygonStyle = new KmlPolygonStyle(polygonColor);
```
Comment (Rachael-E) [inline kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]:
```suggestion
            var kmlPolygonStyle = new KmlPolygonStyle(polygonColor);
```

**Review by Rachael-E — CHANGES_REQUESTED** [review]
@Luiy0 thanks, lots of really great changes here that improve our samples readability so much. Just a couple of comments here, mostly regarding existing code in the samples that we can take the opportunity to improve fairly easily.

File: geometry/geometry-engine-simplify/src/main/java/com/esri/samples/geometry_engine_simplify/GeometryEngineSimplifySample.java [hunk 7]
```diff
@@ -203,7 +204,7 @@ private void createPolygon() {
     polygonParts.add(partThree);
 
     // transparent (0x00000000) fill
```
Comment (Rachael-E) [inline geometry/geometry-engine-simplify/src/main/java/com/esri/samples/geometry_engine_simplify/GeometryEngineSimplifySample.java]:
```suggestion
    // transparent fill
```

**Review by Rachael-E — COMMENTED** [review]
@Luiy0 I did a search for "0x" in IntelliJ for the .java classes and found a couple more references to the old color codes that should be tidied up in this PR. Example of one here: most are in comments, see screenshot below. Thanks! 
 
Screenshot 2022-11-21 at 14 00 23

File: network_analysis/closest-facility-static/src/main/java/com/esri/samples/closest_facility_static/ClosestFacilityStaticSample.java [hunk 8]
```diff
@@ -108,7 +108,7 @@ public void start(Stage stage) throws Exception {
       PictureMarkerSymbol incidentSymbol = new PictureMarkerSymbol(new Image("https://static.arcgis.com/images/Symbols/SafetyHealth/esriCrimeMarker_56_Gradient.png", 30, 30, true, false));
 
       // create a line symbol to mark the route
-      SimpleLineSymbol simpleLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("0x0000FF", 0.4), 5.0f);
+      SimpleLineSymbol simpleLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("blue", 0.4), 5.0f);
```
Comment (Rachael-E) [inline network_analysis/closest-facility-static/src/main/java/com/esri/samples/closest_facility_static/ClosestFacilityStaticSample.java]:
```suggestion
      var simpleLineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.web("blue", 0.4), 5.0f);
```

**Review by Rachael-E — APPROVED** [review]
looks good @Luiy0 thanks! tiny suggestion then good for second review.

**Comment by Luiy0** [comment]
Ready for a second review @jenmerritt, thanks.

File: geometry/geodesic-operations/src/main/java/com/esri/samples/geodesic_operations/GeodesicOperationsSample.java [hunk 9]
```diff
@@ -27,6 +27,7 @@
 import javafx.scene.control.Label;
 import javafx.scene.input.MouseButton;
 import javafx.scene.layout.*;
```
Comment (jenmerritt) [inline geometry/geodesic-operations/src/main/java/com/esri/samples/geodesic_operations/GeodesicOperationsSample.java]: Could clean this `*`

File: kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java [hunk 10]
```diff
@@ -27,7 +27,7 @@
 import com.esri.arcgisruntime.mapping.view.SketchCreationMode;
 import com.esri.arcgisruntime.mapping.view.SketchEditor;
 import com.esri.arcgisruntime.ogc.kml.*;
```
Comment (jenmerritt) [inline kml/create-and-save-kml-file/src/main/java/com/esri/samples/create_and_save_kml_file/CreateAndSaveKMLFileController.java]: could tidy up this `*`

**Review by jenmerritt — APPROVED** [review]
Wow, ok I think this is fine. Couple of import expansions but approving.

**Pull request merged**